specification/eventgrid/data-plane/Microsoft.EventGrid/stable/2018-01-01/EventGrid.json

```diff
@@ -306,6 +306,13 @@
       "description": "Version of the API to be used with the client request.",
       "required": true,
       "type": "string"
+    },
+	"ChannelNameParameter": {
+      "name": "aeg-channel-name",
+      "in": "header",
+      "description": "Required only when publishing to partner namespaces with partner topic routing mode 'ChannelNameHeader'",
+      "required": false,
+      "type": "string"
```

Comment:
You need to add x-ms-parameter location: method. Example:
https://github.com/Azure/azure-rest-api-specs/blob/88e7838a09868a51de3894114355c75929847a46/specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimanagement.json#L161